doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2594,7 +2594,7 @@ NCSIDB-925: Event subscribers in the :ref:`app_event_manager` may overlap when u
 Modem libraries
 ===============
 
-.. rst-class:: v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0
```

Comment:
This was fixed in 2022 (from NCS v2.3.0)
See https://github.com/nrfconnect/sdk-nrf/pull/9563, @jtguggedal to confirm

Reply:
That's correct, it's been fixed since NCS v2.3.0

Reply:
@divipillai this should be updated!